Management/Clients/AirQualityDataClient.cs

```diff
@@ -98,12 +98,14 @@ public AirQualityDataClient(IConfiguration configuration)
             CountryCode countryCode,
             CancellationToken cancellationToken)
         {
-            var sampledCities = await GetDefaultCitiesAsync(state, countryCode, cancellationToken);
+            var abbrStateDict = new AbbrToStateDict();
+            var stateCityDict = new StateToCityDict();
+            var supportedCities = stateCityDict.stateToCityDict[state.Value];
 
             var cityBag = new ConcurrentBag<(City city, int score)>();
-            var cityTasks = sampledCities.Select(async city =>
+            var cityTasks = supportedCities.Select(async city =>
             {
-                var result = await GetSingleCityAsync(city, state, countryCode, cancellationToken);
+                var result = await GetSingleCityAsync(city, abbrStateDict.abbrToStateDict[state.Value], countryCode, cancellationToken);
```

Comment:
Now we directly use the static map instead of calling supported cities from the API.